aiplatform/snippets/embeddings.go

```diff
@@ -28,7 +28,8 @@ import (
 )
 
 func embedTexts(
-	apiEndpoint, project, model string, texts []string, task string) ([][]float32, error) {
+	apiEndpoint, project, model string, texts []string,
+	task string, customOutputDimensionality *int) ([][]float32, error) {
```

Comment:
suggestion: These parameters need additional usage documentation and several should possibly be bundled into a struct. See #4110 and #4113 for examples.

Reply:
SG, I'll hard-coded some of them and only leave `project` and `texts.`